src/csharp/Grpc.Core.Tests/ChannelCredentialsTest.cs

```diff
@@ -30,6 +30,12 @@ public void InsecureCredentials_IsNonComposable()
             Assert.IsFalse(ChannelCredentials.Insecure.IsComposable);
         }
 
+        [Test]
+        public void SecureCredentials_IsNonComposable()
+        {
+            Assert.IsFalse(ChannelCredentials.Secure.IsComposable);
```

Comment:
ssl credentials are actually composable: https://github.com/grpc/grpc/blob/fd3bd70939fb4239639fbd26143ec416366e4157/src/csharp/Grpc.Core.Api/SslCredentials.cs#L118 (since they can be composed with callCredentials: https://github.com/grpc/grpc/blob/fd3bd70939fb4239639fbd26143ec416366e4157/src/csharp/Grpc.Core.Api/ChannelCredentials.cs#L61)

Reply:
Fixed